bundle/Templating/Twig/Extension/SiteExtension.php

```diff
@@ -29,6 +29,10 @@ public function getFunctions(): array
                 'ngsite_location_name',
                 [SiteRuntime::class, 'getLocationName'],
             ),
+            new TwigFunction(
+                'ngsite_image_absolute_url',
```

Comment:
Couple of questions?

Are you sure this returns the absolute URL?

`$this->imageVariationService->getVariation($field->innerField, $content->innerVersionInfo, $alias)->uri` doesn't look like it returns an absolute url? If not, why not just `ngsite_image_url` ?

If it does return an absolute URL, where is it used? Can't you use `url(asset(...))` ? instead?

Reply:
You might be right about the first part.

For the second part, this is introduced to work with both ezimage and ngremotemedia, so other twig functions would not work in this case. (unless we introduce if-else in the template instead - this seemed cleaner to me at the time).

Reply:
:+1: on the second part

Reply:
Name changed in 53b9b0c9998bb1518a2495ef56c63182b74c2e24